docs/src/cli_usage/examples.md

```
@@ -2,97 +2,162 @@
+
+   ```bash
+   rustree -L 1 --report-sizes ./src --llm-ask "What are the main components in the src directory based on this tree?"
+   ```
+
+   Then, you would typically pipe this entire output to your LLM command-line tool. For example:
+
+   ```bash
+   rustree -L 1 --report-sizes ./src --llm-ask "Summarize these components" | ollama run mistral
+   ```
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)